net.mkiol.Jupii.yaml

```diff
@@ -0,0 +1,246 @@
+app-id: net.mkiol.Jupii
+runtime: org.kde.Platform
+runtime-version: "5.15-23.08"
+sdk: org.kde.Sdk
+rename-icon: jupii
+command: jupii
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --device=all # v4l2 webcams
+  - --filesystem=xdg-desktop
+  - --filesystem=xdg-download
+  - --filesystem=xdg-music
+  - --filesystem=xdg-documents
+  - --filesystem=xdg-videos
+  - --talk-name=org.freedesktop.Notifications
```

Comment:
(Line 19 only) is this really necessary? Modern toolkit handle the notification portal properly.

Reply:
Unfortunately it is. This app makes a direct dbus call to trigger the notification. I will try to change this in the future, but currently it works like this.